src/data/dfinsupp/basic.lean

```diff
@@ -529,6 +529,12 @@ begin
     { rw [hi, hj, dfinsupp.single_zero, dfinsupp.single_zero], }, },
 end
 
+/-- `dfinsupp.single a b` is injective in `a`. For the statement that it is injective in `b`, see
+`dfinsupp.single_injective` -/
```

Comment:
Should `dfinsupp.single_injective` be called `dfinsupp.single_right_injective` now?

Reply:
`dfinsupp.single_injective` matches `finsupp.single_injective` and `pi.single_injective` and all of these occur in a lot of places. I'm not opposed to changing this, but I'd rather do it in a follow-up PR.